docs/topics/numbers.md

```diff
@@ -24,6 +24,8 @@ val oneLong = 1L // Long
 val oneByte: Byte = 1
 ```
 
+In addition to integer types, Kotlin also provides unsigned integer types. For more information, see [Unsigned integer types](unsigned-integer-types.md).
```

Comment:
Below the link to the same page is designed as a note, I think we should make it consistent

Reply:
Good point @danil-pavlov ! I converted it to a tip to make it consistent.